spec/acceptance/basic_candlepin_spec.rb

```diff
@@ -9,7 +9,25 @@
     it { is_expected.to be_listening }
   end
 
-  describe command('curl -k -s -o /dev/null -w \'%{http_code}\' https://localhost:8443/candlepin/status') do
-      its(:stdout) { should eq "200" }
+  describe command("curl -k -s -o /dev/null -w '%{http_code}' https://localhost:8443/candlepin/status") do
+    its(:stdout) { should eq "200" }
+  end
+
+  describe command("curl -s -o /dev/null -w '%{http_code}' --key /etc/candlepin/certs/client-cert.key --cert /etc/candlepin/certs/client-cert.pem --cacert /etc/candlepin/certs/candlepin-ca.crt --tlsv1.3 --tls-max 1.3 https://#{fact('fqdn')}:8443/candlepin/status/") do
+    # We don't enable TLSv1.3 by default yet. TLSv1.3 support was added in tomcat 7.0.92
+    # But tomcat 7.0.76 is the latest version available on EL7
```

Comment:
Perhaps that's something that could be determined via facts in `params.pp`? Sounds like a good improvement on EL8.